app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,33 @@
+class LikesController < ApplicationController
+
+  before_action :require_login
+
+  def create
+    post = Post.find(params[:post_id])
+    like = post.likes.new
```

Comment:
I'm curious as to what post.likes.new actually *does*

I used something like Like.create(params).errors.any? to see if the creation was successful.